Error during shutdown: unsetInbox method not found

Seeing this at shutdown.

`
2016-08-23 07:52:57.970 [ERROR] [g.eclipse.smarthome.config.discovery] - [org.eclipse.smarthome.config.discovery.internal.DiscoveryServiceRegistry(7)] unbind method [unsetInbox] not found; Component will fail
`

Method in `DiscoveryServiceRegistryImpl` is actually called removeInbox.

Assuming that this is because the latest zwave binding uses the new extended discovery services added here.  https://github.com/eclipse/smarthome/pull/1985

--- a/bundles/config/org.eclipse.smarthome.config.discovery/src/main/java/org/eclipse/smarthome/config/discovery/internal/DiscoveryServiceRegistryImpl.java
+++ b/bundles/config/org.eclipse.smarthome.config.discovery/src/main/java/org/eclipse/smarthome/config/discovery/internal/DiscoveryServiceRegistryImpl.java
@@ -474,7 +474,7 @@ public final class DiscoveryServiceRegistryImpl implements DiscoveryServiceRegis
         this.inbox = inbox;
     }
 
-    protected void removeInbox(Inbox inbox) {
+    protected void unsetInbox(Inbox inbox) {
         this.inbox = null;
     }
 

